### DISCUSSION TITLE
Is there anyform of IPC implemented?

### DISCUSSION THREAD

**Discussion by SaverinOnRails** [librespot-org/librespot#1397] [General]
Hello, thank you for this project. I was wondering if there i any recommender way of controlling a running instance of librespot started on a shell without killing it and restarting. Perhaps to locally pause playback, seek,  change volume , e.t.c without using spotify's playback apis? Or would we have to impplement that ourselves in rust. Thanks a lot.

**Comment by roderickvd** [comment]
Librespot doesn't have that. I think there are some other OSS projects that do (spotcontrol?)

**Comment by kingosticks** [comment]
You could try https://github.com/librespot-org/librespot/pull/1341

**Reply by roderickvd** [reply]
> You could try https://github.com/librespot-org/librespot/pull/1341 

Good one.
